### PR TITLE
better explanations and using deer.rerum.io

### PR DESCRIPTION
**Pull request by cubap**

### CODE REVIEW

**Comment by cubap** [comment]
waiting for @thehabes

File: index.html [hunk 1]
```diff
@@ -170,7 +175,7 @@ <h2>Interactions</h2>
 
             <h4>Using <code>deer-link</code></h4>
             <p class="col-12">In any <code>&lt;deer-view></code> element, the <code>deer-link</code> attribute instructs the templates how to attach links to the rendered object. For lists, each item with an <code>@id</code> property will include an anchor tag hyperlinking
-                to the <code>deer-link</code> URL joined with the <code>@id</code>. This example simply changes the hash on the current page.
+                to the <code>deer-link</code> URL joined with the <code>@id</code>. This example simply changes the hash on the current page. Be sure to be aware of this default behavior if you are using a SPA framework that relies on hashes for routing and consider "Listening to Events" instead.
```
Comment (thehabes) [inline index.html:178]: Maybe just `Be aware`.  `Be sure to be aware` hits my ears wrong.

File: index.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
             <h2>Installing DEER</h2>
             <p class="col-12">To install the script on your project, use the <code>deer.js</code> on your page.</p>
             <small class="col-12">
-                <pre>&lt;script src="//centerfordigitalhumanities.github.io/deer/releases/alpha-0.11/deer.js" type="module">&lt;/script></pre>
+                <pre>&lt;script src="//deer.rerum.io/DEER/releases/alpha-0.11/deer.js" type="module">&lt;/script></pre>
```
Comment (thehabes) [inline index.html:44]: Oh, nice!  Are we going to make this our CDN?  Like can I do this in the deer-lr code?

File: index.html [hunk 3]
```diff
@@ -309,7 +314,11 @@ <h2>Forms</h2>
                 <header>Listening to the Form</header>
                 <div class="col deer-view" deer-listening="formExample" deer-template="entity"></div>
             </div>
-
+<p>
+    There are many more complex ways to deal with forms to manage multiselects, arrays, and other types of
```
Comment (thehabes) [inline index.html:318]: Hmm maybe clean this up a bit, I am not entirely sure what you are trying to say.  Maybe something like 

`There are many more complex ways forms acquire data through UI to accommodate more complex data objects that you may want to record.  How to process data from customized UIs is not well-documented yet, but we have many examples in our public code that we can direct you to upon request.`

**Review by thehabes — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (cubap) [reply index.html:44]: I think that is reasonable. It is how we tell people to link it on the site. Also, we will need to remember to put it up on other CDNs when we hit 1.0 for realz.

File: index.html (hunk 3 above)
Comment (cubap) [reply index.html:318]: You write like a man who has slept. I accept this change.

File: index.html (hunk 1 above)
Comment (cubap) [reply index.html:178]: 👂 
🌾 is "Ear of Rice". That's a real thing.

**Pull request merged**